yarn.lock

```diff
@@ -19842,6 +19847,11 @@ regjsparser@^0.6.4:
   dependencies:
     jsesc "~0.5.0"
 
+regression@^2.0.1:
+  version "2.0.1"
+  resolved "https://artifactory.spotify.net/artifactory/api/npm/virtual-npm/regression/-/regression-2.0.1.tgz#8d29c3e8224a10850c35e337e85a8b2fac3b0c87"
```

Comment:
Ah, looks like this made it's way here. I think you want to make sure the npm registry is correctly set when adding packages. `YARN_REGISTRY=https://registry.npmjs.org yarn add ...`

Reply:
Ah, good catch. There's an `.npmrc` with the standard registry in root and our plugin. Did I run yarn install from elsewhere? A mystery for the ages..